### PR TITLE
Add details to getting started

### PR DESCRIPTION
**Pull request by akegan**
Just made a couple of small changes to getting started for the webapp. 

@chrisdolendo @callaghanc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Before you install DAHLIA, your system should have the following:
     - see [here](https://stackoverflow.com/a/19850273/260495) if you have issues installing `pg` gem with Postgres.app, you may need to use: `gem install pg -v 0.18.4 -- --with-pg-config=/Applications/Postgres.app/Contents/Versions/latest/bin/pg_config
 `
 1. `npm install` to install bower, grunt and other dependencies (which will also automatically `bower install` to load front-end JS libraries)
-1. `overcommit --install` to install git hooks into the repo
+1. Run `gem install overcommit` then `overcommit --install` to install git hooks into the repo
```
Comment (akegan) [inline README.md]: looking at the gemfile, this should be there, but it's not getting installed in bundle install for me in this repo or in the LAP.

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ See [here](https://www.lucidchart.com/documents/view/53cd191b-3ca5-4b23-832d-28a
 ## Dependencies
 Before you install DAHLIA, your system should have the following:
 
-- [Ruby](https://www.ruby-lang.org/en/documentation/installation/) 2.2.3
+- [Ruby](https://www.ruby-lang.org/en/documentation/installation/) 2.2.3 (Use [RVM](https://rvm.io/rvm/install) or [rbenv](https://github.com/rbenv/rbenv))
 - [Bundler](https://github.com/bundler/bundler) `gem install bundler -v 1.12.5`
```
Comment (akegan) [inline README.md]: I got a warning with this version saying "Warning: the running version of Bundler is older than the version that created the lockfile. We suggest you upgrade to the latest version of Bundler by running `gem install bundler`."

Is it OK to update it to just `gem install bundler`?

File: README.md (hunk 1 above)
Comment (akegan) [reply README.md]: I uninstalled all gems then re-installed bundle and re-ran bundle install and overcommit installed successfully this time. I went ahead and removed the extra text

File: README.md (hunk 2 above)
Comment (callaghanc) [reply README.md]: Yep go for it!

**Pull request merged**